### DISCUSSION TITLE
Access to the action runs of a repository

### DISCUSSION THREAD

**Discussion by Vampire** [community/community#47667] [Mobile]
### Select Topic Area

Product Feedback

### Body

Currently the Android app has no ability to display the action runs of a repository.
I has the general ability to show action runs, because you can get to it from a pull request.
But if you want to list and look at the list of executed actions, or trigger a manual action, you have to switch to the browser instead.

It would be nice if this also worked from the Android app.

**Comment by candyho** [comment]
Hey, thanks for your feedback! Adding Actions support to a Repo level is something on our roadmap. Please stay tuned for more update in the future.
